### DISCUSSION TITLE
Question display tdb XFA DDt4all

### DISCUSSION THREAD

**Discussion by Newbilou** [cedricp/ddt4all#889] [Q&A]
Hello everyone, excuse my English (google), 
I don't know where to look (on specialized forums, no one answers), 
I'm starting.

My (scenic) XFA no longer displays the distance between vehicles on the TDB. It does not have ACC (no button). Can you tell me what to reactivate on the TDB Cluster to reactivate this information? DW On, AEBS On,

Thank you for your help

**Comment by KarelSvo** [comment]
just like that or after playing with ddt4all?

**Reply by Newbilou** [reply]
Playing with DDT4all but not alone but I thought I had come across someone who knew (now I assure alone but I lack skill and knowledge)
Well, in the first attempt, the knowledgeable person activated Radnav (it seems to me and without making a backup), no pretty Christmas tree but the will of the vehicle to detect ghost vehicles and the car was trying to slow down on its own, after disabling ACC via Radnav, the distance icon never appeared again but the emergency braking option also seems to be broken (no error indication, however)

Finally, he wanted to activate the LKA (which I never activated, not reassured) and touched the ACC, beautiful blunder...

**Comment by KarelSvo** [comment]
If the log files from all ddt4all sessions are there, you or i can search for the original configuration and try to restore from there.

**Reply by Newbilou** [reply]
Good morning,

Given the refresh of our relationship, when his intervention gave this result, 
I am not able to restore with the logs (I don't have them) however, it is likely that someone knows the ECU concerned ( what I haven't found for 4 months on the internet and various forums) to be able to restore the similar configuration?
